### ISSUE TITLE
Unable to import ECC Key

### ISSUE THREAD

**Issue by gprossliner** [Legrandin/pycryptodome#133]
Hello!

Perfekt library, works as expected on the ~~first~~ second try. The only thing I struggled with, was reading my keyfile, generated by openssl with:

`openssl ecparam -name prime256v1 -genkey`

This creates a new EC keypair. You can see two boundaries ("EC PARAMETERS" and "EC PRIVATE KEY") within the file. This doesn't work with the following code:

```
from Crypto.Signature import DSS
key=DSS.import_key(open("key.pem").read())
```

It raises "Not a valid PEM post boundary". I debugged into the source, and it seems that, you expect the file only to have a single boundary. So the condition "EC PARAMETERS" == "EC PRIVATE KEY" fails.

If the first section "EC PARAMETERS" is dropped from the file, leaving only the "EC PRIVATE KEY" there, all is good.

IMHO the minimum fix would be, that no Error is raised on this file.
What would be great, if the EC PARMETERS are actually read, and compared that the correct curve (OID: prime256v1) has been used. 

Is this a known bug?
Thank you.

**Comment by gprossliner** [comment]
I have run the SelfTest with `python setup.py test`. Some tests failed on my machine (Win10, python 2.7), with "nearly true" Assertions. Only \r\n != \n.

```
FAIL: test_export_openssh (Crypto.SelfTest.PublicKey.test_import_ECC.TestExport)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "build/lib.linux-x86_64-2.7/Crypto/SelfTest/PublicKey/test_import_ECC.py", line 310, in test_export_openssh
    self.assertEquals(key_file, encoded)
AssertionError: 
'ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBKQK1ZogUOvpJHm9X7FrsuRbZGXrPLKx7/5CP6vmy3Qk24IZ7wurgKzyb9cFlbYf5HYNM+7YDdA9L9DfsnuM51w=\r\n' != 
'ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBKQK1ZogUOvpJHm9X7FrsuRbZGXrPLKx7/5CP6vmy3Qk24IZ7wurgKzyb9cFlbYf5HYNM+7YDdA9L9DfsnuM51w=\n'
```
Is there a setting to make this work (on Windows too)?

**Comment by Legrandin** [comment]
The "EC PARAMETERS" section is redundant because the content of "EC PRIVATE KEY" is intended to be self-contained (once upon a time, it was possible to have EC private keys defined on an implicit curve).

For sake of simplicity, I would rather prefer to just ignore the "EC PARAMETERS" section, just to deal with this quirk of openssl.

**Comment by gprossliner** [comment]
Hello. I ignore this by now with a regex, before the import_key is called. This is fine for my by now. But nevertheless, I would expect a "graceful" response, without preprocessing.

I also thought about fixing this myself, but the 1:1 relationship between file and key is not that easy to work around.

Thank you again for this wonderful library.

**Comment by Legrandin** [comment]
Fixed on master with b9f41b40a6d239be35caf27aad0bee2a66b87cf5.

**Comment by Legrandin** [comment]
Released in v3.8.1

**Comment by gprossliner** [comment]
Hey!

Thank you for implementing the enhancement. The test reminds me of b10d9ac ;-)
Now I will update and remove my regex preprocessing.

GP
